src/api/app/views/webui/projects/maintained_projects/index.html.haml

```diff
@@ -0,0 +1,37 @@
+:ruby
+  @pagetitle = "Projects Maintained by #{@project}"
+  update_policy = policy(@project).update?
+
+.card
+  .card-body
+    - if update_policy
+      %ul.list-inline
+        %li.list-inline-item.mr-3
+          = link_to('#', data: { toggle: 'modal', target: '#new-maintenance-project-modal' }) do
+            %i.fas.fa-plus-circle.text-primary
+            Add New Project
```

Comment:
`Add New` is redundant. We should use `Add Project` or perhaps, `Maintain a Project`.